fundamental-concepts/data-structures-stdlib/3_sums.md

```diff
@@ -0,0 +1,35 @@
+# Enumeration
+
+The next fundamental data structure is the enumeration:
+
+<pre class="file" data-filename="daml/AddressBook.daml" data-target="append">
+data Gender
+  = M
+  | F
+</pre>
+
+Each option is named with an upper-case constructor and preceded by a `|`. An enumeration is just a
+special case of a sum data type:
+
+<pre class="file" data-filename="daml/AddressBook.daml" data-target="append">
+  | Other Text
+</pre>
+
+Here, we added a third constructor with a text field. Sum data types can have any number of
+constructors listed with a preceding `|`. Every constructor can take one or more arguments.
+
+When you want to **deconstruct** a sum type in a function, you need to use a **case** statement:
+
+<pre class="file" data-filename="daml/AddressBook.daml" data-target="append">
+prettyGender: Gender -> Text
+prettyGender gender = case gender of
+  M -> "male"
+  F -> "female"
+  Other o -> o
+</pre>
```

Comment:
Just writing here my thoughts as they happen while going through the scenario:

Is `sum` the right word here? While reading I was expecting to sum up numbers, e.g., sum 1, 2, 3, 4 returning 10.

Reply:
Yes, these are `sum` types, while records, tuples are `product` types.